backbone/src/main/java/org/researchstack/backbone/model/survey/factory/SurveyFactory.java

```diff
@@ -111,6 +111,17 @@ public SurveyFactory() {
     }
 
     /**
+     * Creates a Step which is not a subtask step.
+     *
+     * @param context can be any context, activity or application, used to access "R" resources
+     * @param item the survey item to act upon
+     * @return a step created from the item
+     */
+    public Step createSurveyStep(Context context, SurveyItem item) {
+        return createSurveyStep(context, item, false);
```

Comment:
I have this fixed on another branch too, my bad on this.  I hope it didn't cause you too much confusion.